package/yast2-metapackage-handler.changes

```diff
@@ -1,6 +1,12 @@
 -------------------------------------------------------------------
 Mon Nov  3 14:45:57 UTC 2014 - lslezak@suse.cz
```

Comment:
Just use the current date and your email, be proud for your contribution! :wink: 

Just run `osc vc`, it will create the required line correctly.